docs/pywbemclisubcommands.rst

```diff
@@ -45,6 +45,190 @@ Command Details`
 (``-m``\``--mock-server`` general option) with mock files in the pywbemtools
 tests/unit subdirectory to generate known results.
 
+.. _`Pywbemcli special command line features`:
+
+Pywbemcli special command line features
+---------------------------------------
+
+Pywbemcli includes several features in the command syntax that are worth
+presenting in detail to help the user understand the background purpose and
+syntactic implementation of the features. This includes:
+
+* The ability to execute either the pull or traditional operations with the
+  same command group.
+
+* The ability to receive either CIM instances or CIM instance names with only
+  a change of an option on the commands that request CIM instances. The option
+  ``-o`` or ``--name-only`` defines whether only the instance name or the complete
```

Comment:
Update option name `--name-only` to `--names-only`.

Reply:
DONE

Reply:
Scanned whole doc directory for name-only and found no more.